src/components/share-overlay/share-overlay.js

```diff
@@ -388,6 +439,7 @@ class ShareOverlay extends Component {
                 startFromValue={this.state.startFromValue}
                 handleStartFromChange={this._handleStartFromChange}
                 toggleStartFrom={this._toggleStartFrom}
+                videoHasHours={this.props.player.duration >= 3600}
```

Comment:
what if it's live ?

Reply:
we do not render this component if it's live- see line we do not render this component if it's live- see line #435